applications/ContactStructuralMechanicsApplication/custom_conditions/ALM_frictionless_mortar_contact_condition.h

```diff
@@ -232,7 +232,7 @@ class AugmentedLagrangianMethodFrictionlessMortarContactCondition: public Augmen
     bounded_matrix<double, MatrixSize, MatrixSize> CalculateLocalLHS(
         const MortarConditionMatrices& rMortarConditionMatrices,
         const DerivativeDataType& rDerivativeData,
-        const unsigned int& rActiveInactive
+        const unsigned int rActiveInactive
```

Comment:
just as a comment: it doesn't sound as a very good name. should it be a bool?

Reply:
It is a int because is the total combination of active inactive nodes, so for example a full active triangle will be 111,that in binary is 8. I will add the proper explanation